services/src/main/java/org/fao/geonet/api/site/SiteInformation.java

```diff
@@ -226,6 +226,13 @@ private void loadDatabaseInfo(ServiceContext context) throws SQLException {
             connection = context.getBean(DataSource.class).getConnection();
             dbURL = connection.getMetaData().getURL();
             databaseProperties.put("db.openattempt", "Database Opened Successfully");
+            databaseProperties.put("db.product", connection.getMetaData().getDatabaseProductName());
```

Comment:
I would call that `db.type`, to be consistent with the other places this can be defined